mod-orders/examples/bindPiecesCollection.sample

```diff
@@ -0,0 +1,26 @@
+{
+  "toBeBound": [
```

Comment:
In the AC of releated UI story written
![image](https://github.com/folio-org/acq-models/assets/25097693/a6038cb8-380c-4eac-bc62-890fb8df44b9)
And the same title has the same poLined so we don't need possibility to specify multiple poLineIds at the same time.
Need to think how to simplify the contract

Reply:
other services use the same structure, so you can ignore this comment

Reply:
Ok, thanks to clarification